src/components/Onboarding/steps/GenuineCheck/GenuineCheckUnavailable.js

```diff
@@ -51,7 +58,18 @@ export function GenuineCheckUnavailableMessage({
 }) {
   return (
     <Box align="center" flow={1} color={colors.alertRed}>
-      <FakeLink ff="Open Sans|Regular" fontSize={4} underline onClick={handleOpenGenuineCheckModal}>
+      <FakeLink
+        ff="Open Sans|Regular"
+        fontSize={4}
+        underline
+        onClick={() => {
+          handleOpenGenuineCheckModal()
+          track('Genuine Check Unavailable Retry', {
```

Comment:
maybe just 'Genuine Check Retry' ? because if you retry, it's because it was unavailable

Reply:
yes, also it is more general. Retry is also the case if you disattach the device in the middle